pkg/v1/providers/config_default.yaml

```diff
@@ -111,6 +111,10 @@ VSPHERE_PASSWORD:
 VSPHERE_REGION:
 VSPHERE_ZONE:
 
+VSPHERE_AZ_0:
+VSPHERE_AZ_1:
+VSPHERE_AZ_2:
+
```

Comment:
Question: Do we have any validation against above three variables ? I am assuming only non-empty is allowed, correct ? Or maybe the validation has been done by UI already.

Reply:
Question: Do we have a PR for UI accordingly ?

Reply:
Good questions! This is for workload clusters, so there are no UI changes required. 

These variables can be empty as well, because vSphere did not support AZs until now, so if some does not want to specify AZs, then they should be allowed to do so.